atd-etl/app/process/helpers_import.py

```diff
@@ -663,19 +665,21 @@ def record_crash_compare(line, fieldnames, crash_id, record_existing):
         else:
             print(differences)
 
-        # If no changes approved, or longer than 3 days
+        # check if record is at least 3 days old
         if can_update is False:
             # It's too soon, ignore change.
             return False, "Cannot update this record (Record is not older than 3 days)"
 
-        # There are differences, and it can update unless
+        # There are differences, and it can automatically update unless
         # either human_updated or important_update is true
         compare_enabled = (
             ATD_ETL_CONFIG["ATD_CRIS_IMPORT_COMPARE_FUNCTION"] == "ENABLED"
         )
-        if human_updated or important_update:
+        if human_updated:
+          return False, "Skipping changes on manually QA/updated record"
+        elif important_update:
```

Comment:
Here is my change in accordance with the issue: 

> So that the query that populates this table checks whether the status is manually QA-ed and blocks incoming changes so it doesn't show up on the table.

Reply:
> Accept changes to lat/long changes by default (if not manually updated previously), and don't include those records in this index.

lat/long isn't included as one of the important fields when checking for important update.

Reply:
> > Accept changes to lat/long changes by default (if not manually updated previously), and don't include those records in this index.
> 
> lat/long isn't included as one of the important fields when checking for important update.

I think those two fields may be actually included in the list of important fields; [line 355 & 356](https://github.com/cityofaustin/atd-vz-data/blob/4582-crash-logic/atd-etl/app/process/helpers_import_fields.py#L355-L356) in `helpers_import_fields.py`.

I think they may need to be pulled out and handled with a special case bit of logic in [helpers_import.py](https://github.com/cityofaustin/atd-vz-data/blob/4582-crash-logic/atd-etl/app/process/helpers_import.py#L509).

Reply:
They are included in the list of differences to look for, but the important fields just checks for one of two: https://github.com/cityofaustin/atd-vz-data/blob/4582-crash-logic/atd-etl/app/process/helpers_import.py#L630